src/django/api/aws_batch.py

```diff
@@ -176,6 +175,58 @@ def append_processing_result(result_dict):
         'is_array': False,
         'started_at': started,
         'finished_at': finished,
-    })
+    }, facility_list)
+
+    return job_ids
+
+
+def submit_parse_job(facility_list):
+    """
+    Submit AWS Batch job to parse each FacilityListItem in a FacilityList.
+    https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/batch.html#Batch.Client.submit_job
+    """
+    client = boto3.client('batch')
+    job_data = {
+        'client': client,
+        'queue_arn': fetch_batch_queue_arn(client),
+        'job_def_arn': fetch_latest_active_batch_job_definition_arn(client),
+        'job_time': (timezone.now().isoformat()
+                     .replace(':', '-')
+                     .replace('.', '-')
+                     .replace('T', '-')
+                     .replace('+', '-')),
+        'facility_list': facility_list,
+    }
+
+    return parse(job_data, facility_list)
+
+
+def submit_jobs(facility_list, job_ids=None):
+    """
+    Submit AWS Batch jobs to process each FacilityListItem in a FacilityList
+    through the geocode, match, and notify list processing steps.
+    https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/batch.html#Batch.Client.submit_job
+    """
+    client = boto3.client('batch')
+    job_data = {
+        'client': client,
+        'queue_arn': fetch_batch_queue_arn(client),
+        'job_def_arn': fetch_latest_active_batch_job_definition_arn(client),
+        'job_time': (timezone.now().isoformat()
+                     .replace(':', '-')
+                     .replace('.', '-')
+                     .replace('T', '-')
+                     .replace('+', '-')),
+        'facility_list': facility_list,
+    }
+
+    if job_ids is None:
+        job_ids = []
+
+    geocode(job_data, facility_list, job_ids)
+
+    match(job_data, facility_list, job_ids)
+
+    notify(job_data, facility_list, job_ids)
```

Comment:
Consider adding a comment that makes it explicit that these calling these functions mutate the `job_ids` parameter.

Reply:
Good call - updated!